src/components/AppNavigation/Settings.vue

```diff
@@ -218,7 +218,7 @@ export default {
 			return this.$store.state.importState.importState.stage === IMPORT_STAGE_IMPORTING
 		},
 		settingsTitle() {
-			return this.$t('calendar', 'Settings & import').replace(/&amp;/g, '&')
+			return this.$t('calendar', 'Calendar Settings')
```

Comment:
```suggestion
			return this.$t('calendar', 'Calendar settings')
```
We use "Sentence case" instead of "Capital Case" as per design guidelines. :)

Reply:
dammit, that reflex again :-(